learn/using-contracts/cw20.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Sending CW20 Tokens
+order: 2
+---
+
+Now that you have set up your client, let's get going and try out the
+[`cw20-base`](https://github.com/CosmWasm/cosmwasm-plus/tree/master/contracts/cw20-base)
+token contract, which implements the
+[`cw20` spec](https://github.com/CosmWasm/cosmwasm-plus/blob/master/packages/cw20/README.md).
+
+We will show how to upload the compiled wasm code, instantiate your own contract (possibly
+reusing code), and then mint and transfer tokens on that contract.
+
+## Upload and Instantiate a Contract
+
+I will walk you though how to set up an example CW20 contract on CoralNet.
+
+### Example: STAR
+
+The first contract I uploaded was STAR tokens, or "Golden Stars" to be distribute to the
+[first 7 validators](https://bigdipper.coralnet.cosmwasm.com/validators) on the network.
+
+Please do not copy this verbatum, but look to see how such a contract is setup and deployed the first time.
+
+```ts
+const client = await useOptions(coralnetOptions).setup(YOUR_PASSWORD_HERE);
+
+const cw20 = CW20(client);
+const codeId = await cw20.upload();
+console.log(`CodeId: ${codeId}`);
+// output: 4
+
+.editor
+const initMsg: InitMsg = {
+  name: "Golden Stars",
+  symbol: "STAR",
+  decimals: 2,
+  // list of all validator self-delegate addresses - 100 STARs each!
+  initial_balances: [
+    { address: "coral1exta8hzrghyt5umd4jh55kfkmp0tv3hyg8krc5", amount: "10000"},
+    { address: "coral13mcejut8e5tncs59zcs4yn4envcd98vx682frk", amount: "10000"},
+    { address: "coral10zn0d2eeust0495crtr3zqz7t688hg0s53afrh", amount: "10000"},
+    { address: "coral1qvrcashqpemlkhrqphzv9n5nutdxpafmdefgcl", amount: "10000"},
+    { address: "coral14f8nvyy4c9pyn78dgv0k6syek3jjjrkyz747kj", amount: "10000"},
+    { address: "coral1e86v774dch5uwkks0cepw8mdz8a9flhhapvf6w", amount: "10000"},
+    { address: "coral1hf50trj7plz2sd8cmcvn7c8ruh3tjhc2nhyl7l", amount: "10000"},
+  ],
+  mint: {
+    minter: client.senderAddress,
+  },
+};
+^D
+
+const contract = await cw20.instantiate(codeId, initMsg, "STAR");
+console.log(`Contract: ${contract.contractAddress}`);
+// Contract: coral16t7y0vrtpqjw2d7jvc2209yan9002339mg4mrv
+
+console.log(await contract.balance("coral13mcejut8e5tncs59zcs4yn4envcd98vx682frk"));
+// 10000
+console.log(await contract.balance());
+// 0
+```
+
+If you want to see this is real, just check out
+[Code ID 4](https://coralnet.wasm.glass/#/codes/4)
+and [contract `coral16t7y0vrtpqjw2d7jvc2209yan9002339mg4mrv`](https://coralnet.wasm.glass/#/contracts/coral16t7y0vrtpqjw2d7jvc2209yan9002339mg4mrv)
+on the wasm.glass code explorer.
+
+### Try this at home: MINE
+
+Now that we have that uploaded, we can easily make a second contract. This one, please
```

Comment:
I found great vuepress gems today. https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers. We can use warning, tip containers to point out warning. I'll go through the documentation after this PR and add containers.

Reply:
Ah, cool. Happy for any design improvements.

I would merge this PR once I address the comments, then let you add another PR to make it cleaner.